lib/diagnostic.rb

```diff
@@ -93,13 +95,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.pop arr.pop'
```

Comment:
Good, we can also do `arr.pop 2` to delete two things from the end of the array.